__tests__/app.test.js

```diff
@@ -345,26 +353,82 @@ describe("11. GET /api/articles topic query", () => {
+    });
+  });
+
+  test("Returns array of objects ", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((topic) => {
+          expect(topic).toMatchObject({
+            "article_id": expect.any(Number),
+            "title": expect.any(String),
+            "topic": expect.any(String),
+            "author": expect.any(String),
+            "body": expect.any(String),
+            "created_at": expect.any(String),
+            "votes": expect.any(Number),
+            "article_img_url": expect.any(String),
+            "comment_count": expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("Returns single object for cats ", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          articles: [
+            {
+              article_id: 5,
+              title: 'UNCOVERED: catspiracy to bring down democracy',
+              topic: 'cats',
+              author: 'rogersop',
+              body: 'Bastet walks amongst us, and the cats are taking arms!',
+              created_at: '2020-08-03T13:14:00.000Z',
+              votes: 0,
+              article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+              comment_count: '2'
+            }
+          ]
         });
       });
+  });
+});
+
+describe("12. GET /api/articles/:article_id (comment_count)", () => {
+  describe("/api/articles/:article_id", () => {
+    describe("GET /api/articles/:article_id returns 200", () => {
+      test("status code: 200", () => {
+        return request(app).get("/api/articles/1").expect(200);
+      });
     });
 
-    // test("Returns array of objects ", () => {
-    //   return request(app)
-    //     .get("/api/users")
-    //     .expect(200)
-    //     .then(({ body }) => {
-    //       body.forEach((topic) => {
-    //         expect(topic).toMatchObject({
-    //           username: expect.any(String),
-    //           name: expect.any(String),
-    //           avatar_url: expect.any(String),
-    //         });
-    //       });
-    //     });
-    // });
+    test("Returns object with comment count included ", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: "11",
+          });
```

Comment:
👍  Although -do you need to check all of the properties? Surely you only _need_ to check that the returned article has a comment_count property?